src/common/components/clusterConfiguration/CertificateFields.tsx

```diff
@@ -42,14 +42,6 @@ export const CertificateInputFields = () => {
         helperText={<CertificateFieldsHelperText />}
         idPostfix="discovery"
```

Comment:
`idPostfix=discovery` probably comes from UploadSSH key, we can renemd it for `certificates`